client-v2/src/actions/Collections.ts

```diff
@@ -390,6 +389,18 @@ function publishCollection(
   };
 }
 
+function discardDraftChangesToCollection(
+  collectionId: string
+): ThunkResult<Promise<void | string[]>> {
+  return (dispatch: Dispatch, getState: () => State) => {
+    return discardDraftChangesToCollectionApi(collectionId)
+      .then(() => dispatch(getCollections([collectionId])))
+      .catch(() => {
```

Comment:
Whilst we're waiting for error handling, it might be better to omit this empty `catch` and allow sentry to pick it up.